styles.css

```diff
@@ -538,31 +531,207 @@ h1.heading {
   margin-bottom: 8px;
 }
 
-@media screen and (max-width: 768px) {
-  .container_small {
-    padding-left: 8px;
-    margin-top: -60px;
+.hand {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    background-color: rgb(240, 240, 240);
+  }
+
+  .material {
+    background-color: #fff;
   }
 
   .section {
+    padding: 0 3%;
     background-image: url(asset/header-shapes-mobile.svg);
```

Comment:
- [ ] Kindly use the background image for the desktop version in the Figma design.
#### Figma
![headline-bg-image-figma](https://user-images.githubusercontent.com/44057863/167259127-bf20b1cb-8343-4a80-a5b5-6b4d78f89761.png)
#### Imp
![headline-bg-image-impl](https://user-images.githubusercontent.com/44057863/167259136-fd04bdac-f632-488d-af36-32053ef93c00.png)
l

Reply:
exactly the same image that I am using 
I downloaded the desktop template of the image